### DISCUSSION TITLE
DW-8000 import issues

### DISCUSSION THREAD

**Discussion by ZachDimmick** [christofmuc/KnobKraft-orm#328] [General]
Patches from my first import included seemingly all information except the waveform which incorrectly stays the same as the current patch loaded prior to bank import. Additionally all patches, any bank imported to Orm read as patch 11 in Orm and renaming doesn't transfer back either. Maybe renaming is purely for cataloging? 

This program looks to be indispensable if I can sort those things out, anything obvious I may be missing? Using a UM-ONE midi to usb on Mac. Much appreciated!

**Comment by christofmuc** [comment]
Hi, thanks for the feedback!

Let me see - what do zou mean with Waveform stays the same? You send a patch and one parameter is not updated in the synth? That would be very unusual.

Renaming cannot be tranferred back into the synth, as the DW8000 has no name display and the patch data contains no name. The patch data even contains no number, which is why they import as 11. You can use the Bulk Rename dialog to quickly paste names from the clipboard or use filenames when each patch was stored in a separate file with a meaningful name. 

Sending the patch into the synth should work, and if you don't constuct a new bank then the patch will only be send into the edit buffer, and not saved in the synth as long as you do not store it. You can create banks and drag and drop patches, and then send a bank overwriting all of the synths memory.

**Reply by ZachDimmick** [reply]
Understood, thanks! As for the waveform issues I had modified some patches in the Angel City bank, dumped them into Orm, messed around with some other banks and eventually reloaded the modified AC bank - but yes which ever waveform was used in the DW patch prior to importing this bank remains present amongst all dumped patches. I can’t say for certain if the rest of the patch information is correct but the obvious difference is waveform. Could just be an anomaly as I tried different banks with no issues.

**Comment by christofmuc** [comment]
Closing this as solved.
